presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1358,7 +1358,7 @@ private static Table buildTableObject(
         return tableBuilder.build();
     }
 
-    private static PrincipalPrivileges buildInitialPrivilegeSet(String tableOwner)
+    public static PrincipalPrivileges buildInitialPrivilegeSet(String tableOwner)
```

Comment:
Does it matter that INSERT, UPDATE is not applicable to Delta tables? Should we just create a new method and remove those?

Reply:
Make sense to remove privileges for INSERT, UPDATE, and DELETE  for now Since these are not yet supported via connector. And this can be added later as the support is added